deepinv/tests/test_physics.py

```diff
@@ -497,10 +502,19 @@ def test_phase_retrieval(device):
     physics = dinv.physics.RandomPhaseRetrieval(
         m=500, img_shape=(1, 10, 10), device=device
     )
+    physics2 = dinv.physics.StructuredRandomPhaseRetrieval(
```

Comment:
I would remove this from here and just test the linear operator ``StructuredRandom`` with the rest of the linear operators